src/framework/components/scroll-view/component.js

```diff
@@ -496,6 +506,47 @@ Object.assign(pc, function () {
             }
         },
 
+        // re-enable useInput flag on any descendent that was disabled
+        _enableContentInput: function () {
+            while(this._disabledContentEntities.length) {
+                var e = this._disabledContentEntities.pop();
+                if (e.element) {
+                    e.element.useInput = true;
```

Comment:
I suppose it's possible that the element could have had its `useInput` property set to `false` by something else during the scroll operation, in which case we'd be setting it back to `true` here inadvertently. Perhaps we don't need to worry about that though.